ign-rviz-plugins/include/ignition/rviz/plugins/tf_display.hpp

```diff
@@ -66,4 +66,8 @@ class TFDisplay : public MessageDisplay<tf2_msgs::msg::TFMessage>
 }  // namespace rviz
 }  // namespace ignition
 
+#include <pluginlib/class_list_macros.hpp>  // NOLINT
+PLUGINLIB_EXPORT_CLASS(ignition::rviz::plugins::TFDisplay,
+                       ignition::rviz::plugins::MessageDisplayBase)
+
```

Comment:
I am not sure but, is the pluginlib export required in the header file if I already have it inside `cpp` file?

Reply:
yes, you are right, I didn't see it